static/js/components/RunSummary.jsx

```diff
@@ -0,0 +1,423 @@
+import { observingRunTitle } from './AssignmentForm';
+import { ObservingRunStarList } from './StarList';
+import * as SourceAction from '../ducks/source';
+import * as Action from '../ducks/observingRun';
+import { ra_to_hours, dec_to_hours } from "../units";
+
+const VegaPlot = React.lazy(() => import('./VegaPlot'));
+const AirmassPlot = React.lazy(() => import('./AirmassPlot'));
+
+
+const SimpleMenu = ({ assignment }) => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const dispatch = useDispatch();
+
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const ispending = assignment.status === "pending";
+  const isdone = assignment.status === "complete";
+  const isnotdone = assignment.status === "not observed";
+
+  const completeAction = () => {
+    handleClose();
+    return dispatch(
+      SourceAction.editAssignment({ status: "complete" }, assignment.id)
+    );
+  };
+
+  const pendingAction = () => {
+    handleClose();
+    return dispatch(
+      SourceAction.editAssignment({ status: "pending" }, assignment.id)
+    );
+  };
+
+  const notObservedAction = () => {
+    handleClose();
+    return dispatch(
+      SourceAction.editAssignment({ status: "not observed" }, assignment.id)
+    );
+  };
+
+  return (
+    <div>
+      <IconButton
+        aria-controls="simple-menu"
+        aria-haspopup="true"
+        onClick={handleClick}
+        variant="contained"
+      >
+        <BuildIcon />
+      </IconButton>
+      <Menu
+        id="simple-menu"
+        anchorEl={anchorEl}
+        keepMounted
+        open={Boolean(anchorEl)}
+        onClose={handleClose}
+      >
+        {
+          (ispending || isnotdone) && (
+          <MenuItem onClick={completeAction} variant="contained" key={`${assignment.id}_done`}>
+            Mark Observed
+          </MenuItem>
+          )
+}
+        {
+          (ispending || isdone) && (
+          <MenuItem onClick={notObservedAction} variant="contained" key={`${assignment.id}_notdone`}>
+            Mark Not Observed
+          </MenuItem>
+          )
+}
+        {
+          (isdone || isnotdone) && (
+          <MenuItem onClick={pendingAction} variant="contained" key={`${assignment.id}_pending`}>
+            Mark Pending
+          </MenuItem>
+          )
+}
+        {isdone && (
+        <MenuItem key={`${assignment.id}_upload_spec`} onClick={handleClose}>
+          Upload Spectrum
+        </MenuItem>
+        )}
+        {
+          isdone && (
+          <MenuItem
+            key={`${assignment.id}_upload_phot`}
+            variant="contained"
+            onClick={handleClose}
+          >
+            <Link
+              href={`/upload_photometry/${assignment.obj.id}`}
+              underline="none"
+              color="textPrimary"
+            >
+              Upload Photometry
+            </Link>
+          </MenuItem>
+          )
+}
```

Comment:
```suggestion
        }
```
soon, with forced `prettier`, we won't have to worry about these

Reply:
fixed